paddle/operators/compare_op.cc

```diff
@@ -94,5 +94,14 @@ class CompareOp : public framework::OperatorWithKernel {
 
 REGISTER_LOGICAL_OP(less_than, "Out = X < Y");
 REGISTER_LOGICAL_KERNEL(less_than, CPU, paddle::operators::LessThanFunctor);
+REGISTER_LOGICAL_OP(less_than_equal, "Out = X <= Y");
+REGISTER_LOGICAL_KERNEL(less_than_equal, CPU,
+                        paddle::operators::LessThanEqualFunctor);
+REGISTER_LOGICAL_OP(greater_than, "Out = X > Y");
+REGISTER_LOGICAL_KERNEL(greater_than, CPU,
+                        paddle::operators::GreaterThanFunctor);
+REGISTER_LOGICAL_OP(greater_than_equal, "Out = X >= Y");
```

Comment:
greater_equal

I find that the names in Tensorflow's comparison operators are more clear

https://www.tensorflow.org/versions/r0.12/api_docs/python/control_flow_ops/comparison_operators

Reply:
Even I prefer the names in Tensorflow. However, I kept this because we already had the `less_than` op. Do you think I should refactor all of them to have easy and simple names like Tensorflow?

Reply:
I think that the Tensorflow's name is better, shorter but not confusing.

Reply:
Updated in the latest commit.